lib/rubocop/cop/rails/env_local.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks for usage of `Rails.env.development? || Rails.env.test?` which
+      # can be replaced with `Rails.env.local?`, introduced in Rails 7.1.
+      #
+      # @example EnforcedStyle: local (default)
+      #
+      #   # bad
+      #   Rails.env.development? || Rails.env.test?
+      #
+      #   # good
+      #   Rails.env.local?
+      #
+      # @example EnforcedStyle: no_local
+      #
+      #   # bad
+      #   Rails.env.local?
+      #
+      #   # good
+      #   Rails.env.development? || Rails.env.test?
+      #
+      class EnvLocal < Base
+        extend AutoCorrector
+        extend TargetRailsVersion
+
+        LOCAL_MSG = 'Use `Rails.env.local?` instead.'
+        NO_LOCAL_MSG = 'Use `Rails.env.development? || Rails.env.test?` instead.'
+        LOCAL_ENVIRONMENTS = %i[development? test?].to_set.freeze
+        RESTRICT_ON_SEND = %i[local?].freeze
+
+        minimum_target_rails_version 7.1
```

Comment:
I think that this local? method is in rails from 2021.

https://github.com/rails/rails/commit/47eaf8881c049495ca414496a23127e5952d59f0


So if im not wrong you even can enable this for rails 7 and even in 6.1.7?

Reply:
This Rails' feature is not yet released. And RuboCop Rails follows the Rails official, so user custom backports are not supported.

Reply:
Yeah, I'm not sure I follow. The commit you linked is from two weeks ago and as @koic said, is not released yet.

Reply:
true, i thought that i used in rails 6.1.7 maybe im wrong. sorry.